libs/net/core/Extensions/StringExtensions.cs

```diff
@@ -257,14 +257,20 @@ public static string SanitizeContent(string articleContent)
     {
         int startPos = articleContent.IndexOf("<ARTICLE>", StringComparison.CurrentCultureIgnoreCase);
         int endPos = articleContent.IndexOf("</ARTICLE>", StringComparison.CurrentCultureIgnoreCase);
-        string articleStr = "";
+        string articleStr = articleContent;
```

Comment:
You reassign the value of `articleStr` three times.  This requires a new stack of memory each time.  Is there a way to do this more performant?